source/develop/release-management/features/network/autodefine-external-network.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Autodefine External Network
+category: feature
+authors: Ales Musil
+feature_name: Autodefine External Network
+feature_modules: engine
+feature_status: In Development
+---
+
+# Autodefine External Network
+
+## Summary
+
+This feature extends [Provider Physical Network](https://www.ovirt.org/develop/release-management/features/network/provider-physical-network/) 
+adding support for auto-defined external network connected to physical ones. 
+Engine automatically creates an external network based on physical VM network
```

Comment:
With this feature implemented, when a physical VM network is created, Engine automatically creates an external network based on it, and connects the external network to the physical one. This makes it easier to use the former feature, and helps oVirt to model host networks and VM networks separately.

Reply:
Done